source/userguide/augmentation.rst

```diff
@@ -0,0 +1,103 @@
+can be defined to get the best augmentation results based on the application.
+
+Currently, we support data augmentation through paraphrasing for the following languages: English (en), French (fr), Italian (it), Portugese (pt),
+Romanian (ro), Spanish (es). This is done with the use of two models: the English paraphraser and the multi-lingual paraphraser.
+
+First, we will discuss the configuration needed to initiate an augmentor and then follow it with detailed descriptions of the paraphrasers.
+
+Augmentation Configuration
+--------------------------
+
+The :attr:`DEFAULT_AUGMENTATION_CONFIG` shown below is the default config for an Augmentor.
+A custom config can be included in :attr:`config.py` by duplicating the default config and renaming it to :attr:`AUGMENTATION_CONFIG`.
+
+.. code-block:: python
+
+    DEFAULT_AUGMENTATION_CONFIG = {
+        "augmentor_class": "EnglishParaphraser",
+        "paths": [
+            {
+                "domains": ".*",
+                "intents": ".*",
+                "files": ".*",
+            }
+        ],
+        "path_suffix": "-augmented.txt"
+    }
+
+The following values can be updated in the config to customize it:
+
+``'augmentor_class'`` (:class:`str`, default: :class:`EnglishParaphraser`): The augmentor class that can specified for deciding on the augmentation tool to be used. Currently, we recommend using :class:`EnglishParaphraser` for English language applications and :class:`MultiLingualParaphraser` for all other supported languages.
+
+``'paths'`` (:class:`list`): List of regex based path rules to select files to be augmented. These files become the baseline for the augmentors and are used to generate new data. By default, all training and testing files are augmented.
+
+``'path_suffix'`` (:class:`str`): The default suffix that is appended to the name of the original file being augmented to generate new files with the augmented data.
+
+English Paraphraser
+-------------------
+
+The English paraphraser uses a state-of-the-art text summarization model `PEGASUS <https://ai.googleblog.com/2020/06/pegasus-state-of-art-model-for.html>`_ for generating paraphrases. This model has been fine-tuned for the task of paraphrasing. 
+
+Usage
+^^^^^
+
+.. code-block:: console
+
+    mindmeld augment --app-path <app_path> --lang "en"
+
+In the config for this paraphraser class, the ``'augmentor_class'`` should be set to :class:`EnglishParaphraser`.
+
+
+Multi-Lingual Paraphraser
+-------------------------
+
+The multi-lingual paraphraser in MindMeld uses back-translation as the underlying concept to generate paraphrases. Given an application in one of the supported languages, the forward model translates the current set of queries to English, generating a number of English translations. Next, the reverse model translates each of the English translations into one or more queries in the original language. This results in a paraphrased set of queries in the original language.
```

Comment:
The multi-lingual paraphraser in MindMeld uses machine translation as the underlying concept to generate paraphrases.

Reply:
Same question as in the previous question. I think we should be citing the helsinki-NLP models here?